src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -183,10 +184,11 @@ class Simplex_tree {
    * They are essentially pointers into internal vectors, and any insertion or removal
    * of a simplex may invalidate any other Simplex_handle in the complex,
    * unless Options::stable_simplex_handles == true. */
-  typedef typename Dictionary::iterator Simplex_handle;
+  typedef typename Dictionary::const_iterator Simplex_handle;
 
  private:
   typedef typename Dictionary::iterator Dictionary_it;
+  typedef typename Dictionary::const_iterator Const_dictionary_it;
```

Comment:
CGAL uses names like `Vertex_const_iterator`, i.e. it puts the `const_` later in the name. What do you think, between `Const_dictionary_it` and `Dictionary_const_it`?
(not important for a private name)

Reply:
Renamed on [4c1b1d0](https://github.com/GUDHI/gudhi-devel/pull/1145/commits/4c1b1d0a4c214174f5d14cbeaea751c251950719) and I also added this code convention on [a7ca7d2](https://github.com/GUDHI/gudhi-devel/pull/1145/commits/a7ca7d208dce841cf6313124b77b03265c5e2967)